cdap-hbase-compat-base/src/main/java/co/cask/cdap/data2/transaction/coprocessor/CConfigurationCache.java

```diff
@@ -84,7 +109,14 @@ public void run() {
             try {
               CConfiguration newCConf = cConfReader.read();
               if (newCConf != null) {
+                Configuration newConf = new Configuration();
```

Comment:
It would be a good idea to clear the configuration object before adding entries to it.